src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
+
+import { MovieCard } from '../MovieCard/MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <ul className="movies">
+    {movies.map((movie: Movie) => (
+      <li className="card" data-cy="Movie" key={movie.imdbId}>
+        <MovieCard movie={movie} />
```

Comment:
If element has more than 3 attributes - you need to move them to a new line
```suggestion
    {movies.map((movie: Movie) => (
      <li 
        className="card" 
        data-cy="Movie" 
        key={movie.imdbId}
      >
        <MovieCard movie={movie} />
```